nbs/inspect/code.py

```
@@ -179,14 +167,14 @@ def __init__(
                                     ),
                                 )
                             )
-    
+
                 if attributes:
                     table_rows = []
                     for name, value, attr_type, error in attributes:
```

Ruff (0.15.12)

[warning] Redefining argument with the local name `value`

(PLR1704)